### PR TITLE
turned off the keep flag that prevented to test results

### PR DESCRIPTION
**Pull request by mcenerney1**
added antialiasing keyword

### CODE REVIEW

**Comment by aashish24** [comment]
@mcenerney1 why setting threshold to None?

**Comment by doutriaux1** [comment]
@aashish24 these are my changes (pushed from @mcenerney1 computer). I set it to None to mean "use default threshold" if set to another value then the test will use that value.

File: testing/metrics/diags_test.py [hunk 1]
```diff
@@ -82,6 +83,8 @@ def closeness( self, varname, filename, rtol, atol ):
         return close
     def execute(self, test_str, imagefilename, imagethreshold, ncfiles, rtol, atol):
         print test_str
+        if imagethreshold is None:  # user didn't specify a value
```
Comment (doutriaux1) [inline testing/metrics/diags_test.py]: @aashish24 this is why we have `None`

Comment (aashish24) [inline testing/metrics/diags_test.py]: ah thanks @doutriaux1

**Comment by aashish24** [comment]
@mcenerney1 diagnostics tests are failing. 

492/495 Test #490: diags_test_11 ...........................................***Failed   49.84 sec
        Start 491: diags_test_12
493/495 Test #491: diags_test_12 ...........................................***Failed   15.71 sec

**Pull request merged**